### PR TITLE
 [FRONTEND][ONNX] fixed operator converter for Split in onnx frontend

### PR DESCRIPTION
**Pull request by BobLiu20**
fixed issue #2037 

cc @nishi-t @siju-samuel @yzhliu welcome to review. thanks

### CODE REVIEW

File: nnvm/python/nnvm/frontend/onnx.py [hunk 1]
```diff
@@ -754,7 +771,7 @@ def _get_convert_map(opset):
         'Cast': Cast.get_converter(opset),
         'Reshape': Reshape.get_converter(opset),
         'Concat': Renamer('concatenate'),
-        'Split': AttrCvt('split', {'split': 'indices_or_sections'}),
+        'Split': Split.get_converter(opset),
```
Comment (srkreddy1238) [inline nnvm/python/nnvm/frontend/onnx.py]: Pls add a testcase.

Comment (BobLiu20) [reply nnvm/python/nnvm/frontend/onnx.py]: ONNX Split:  https://github.com/onnx/onnx/blob/master/docs/Operators.md#Split
TVM split: https://docs.tvm.ai/nnvm_top.html#nnvm.symbol.split

We need a converter to convert ```split``` from onnx to ```indices_or_sections``` from tvm.
I don't know what's test case do you need. But I had test it in my onnx model and working well.

Comment (srkreddy1238) [reply nnvm/python/nnvm/frontend/onnx.py]: Good that you tested with your model and it's working well.

Please refer to https://github.com/dmlc/tvm/blob/master/nnvm/tests/python/frontend/onnx/test_forward.py for existing testcases for ONNX.

You could use the split layer attributes from your model for the test case here.

Comment (BobLiu20) [reply nnvm/python/nnvm/frontend/onnx.py]: Add test case done.

**Comment by tqchen** [comment]
Thanks @BobLiu20 @srkreddy1238 this is now merged

**Pull request merged**